customize.dist/translations/messages.pl.js

```diff
@@ -0,0 +1,229 @@
+    out.errorBox_errorExplanation_disconnected = [
+        'Połączenie z serwerem zostało zerwane, możesz odświeżyć połączenie przeładowując stronę lub kontynuować przeglądanie dokumentu ',
+        'klikając poza obszar komunikatu.'
+    ].join('');
+
+    out.common_connectionLost = 'Przerwano połączenie z serwerem';
+
+    out.disconnected = 'Rozłączony';
+    out.synchronizing = 'Synchronizacja';
+    out.reconnecting = 'Wznawianie połączenia...';
+    out.lag = 'Lag';
+    out.readonly = 'Tylko do odczytu';
+    out.anonymous = "Anonimowy";
+    out.yourself = "Ty";
+    out.anonymousUsers = "użytkownicy anonimowi";
+    out.anonymousUser = "użytkownik anonimowy";
+    out.shareView = "URL tylko do odczytu";
+    out.shareEdit = "URL do edycji";
+    out.users = "Użytkownicy";
+    out.and = "i";
+    out.viewer = "czytający";
+    out.viewers = "osoby czytające";
+    out.editor = "edytujący";
+    out.editors = "osoby edytujące";
+
+    out.greenLight = "Wszystkie systemy działają poprawnie";
+    out.orangeLight = "Słabe łącze może wpłynąć na działanie aplikacji";
+    out.redLight = "Zostałeś rozłączony z sesją";
+
+    out.importButton = 'IMPORTUJ';
+    out.importButtonTitle = 'Importuj dokument z pliku lokalnego';
+
+    out.exportButton = 'EXPORTUJ';
+    out.exportButtonTitle = 'Zapisz ten dokument do pliku';
+    out.exportPrompt = 'Jak chciałbyś nazwać swój plik?';
+
+    out.back = '&#8656; Wróć';
+    out.backToCryptpad = '⇐ Wróć do Cryptpad';
+
+    out.userButton = 'UŻYTKOWNIK';
+    out.userButtonTitle = 'Zmień swoją nazwę użytkownika';
+    out.changeNamePrompt = 'Zmień swoją nazwę (Pozostaw puste, by być anonimowym): ';
+
+    out.renameButton = 'PRZEMIANUJ'; //Can be changed to "Zmień nazwę"
+    out.renameButtonTitle = 'Zmień nazwę pod którą plik wyświetli się na stronie głównej';
+    out.renamePrompt = 'Jak chciałbyś nazwać ten pad?';
+    out.renameConflict = 'Inny pad już ma taką nazwę';
+    out.clickToEdit = "Naciśnij by edytować";
+
+    out.forgetButton = 'ZAPOMNIJ';
+    out.forgetButtonTitle = 'Usuń ten dokument z listy wyświetlanej na stronie głównej';
+    out.forgetPrompt = 'Wciskając OK usuniesz ten URL z pamięci lokalnej, jesteś tego pewien?';
+
+    out.shareButton = 'Udostępnij';
+    out.shareButtonTitle = "Kopiuj URL do schowka";
+    out.shareSuccess = 'Pomyślnie skopiowano URL';
+    out.shareFailed = "Wystąpił błąd przy kopiowaniu";
+
+    out.presentButton = 'PREZENTUJ';
+    out.presentButtonTitle = "Otwórz tryb prezentacji";
+    out.presentSuccess = 'Naciśnij ESC aby wyjść z trybu prezentacji';
+    out.sourceButton = 'POKAŻ ŹRÓDŁO';
+    out.sourceButtonTitle = "Opuść tryb prezentacji";
+
+    out.backgroundButton = 'KOLOR TŁA';
+    out.backgroundButtonTitle = 'Zmień kolor tła dla tej prezentacji';
+    out.colorButton = 'KOLOR TEKSTU';
+    out.colorButtonTitle = 'Zmień kolor tekstu dla tej prezentacji';
+
+    out.commitButton = 'ZAPISZ'; //Can be discussed
```

Comment:
Publikuj, Utwórz, Dodaj - are just a few ideas that come into my mind when looking at the button. That's why I'd like to hear someone's opinion.
For now, 2 votes for ZAPISZ.